filters/scheduler/fifo.go

```diff
@@ -15,20 +15,44 @@ const (
 	fifoKey string = "fifo"
 )
 
+type fifoType int
+
+const (
+	fifo fifoType = iota + 1
+	fifoWithBody
+)
```

Comment:
You can use filter name as spec.typ and have withBody bool in filter similar to https://github.com/zalando/skipper/blob/2a202c47a7b0f80091f4492374b5c83c868ea9cb/filters/diag/histogram.go#L56-L57

Reply:
I like the enum style better than string.